Modularize, update, and enhance user focus of the "Architecture" part of Planning_for_Project (#2607)
theforeman/foreman-documentation: guides/common/modules/con_client-operating-systems.adoc

[id="Client-Operating-Systems_{context}"]
= Client Operating Systems

> {Project} can help you manage any kind of operating systems that have clients {Project} can integrate with.

Reviewer: This sentence sounds bit confusing.

For example:

* Operating system installers that can perform unattended installations (such as Anaconda in Fedora and CentOS, or Debian-installer in Debian and Ubuntu)
* Puppet
* Ansible
* OpenSCAP
* OpenSSH
* Other clients where integration is provided by external plug-ins

{Project} is actively tested with the following client operating systems:

* CentOS 7 and 8
* Debian stable
* Ubuntu LTS

The Katello plug-in provides functionality for content and subscription management.
The following utilities are provided for selected CentOS and SLES operating systems:

* Subscription manager
* Katello host tools

> * Tracer utility

Reviewer: Alphabetical order?

> For Red Hat family operating systems, SELinux must not be set to disabled mode.

Reviewer: ```suggestion
For Red Hat family operating systems, SELinux must be set to enforcing or permissive mode.
```
Can this be better?
